Header header

float64 S_Ax
float64 S_Ay
float64 S_Az

float64 S_Mx
float64 S_My
float64 S_Mz

float64 S_Gx
float64 S_Gy
float64 S_Gz

uint8 number
uint8 total_number